package.json

```diff
@@ -103,5 +103,8 @@
     "url-parse": "^1.5.0",
     "y18n": "^4.0.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@fluentui/react": "^8.75.1",
```

Comment:
These should be in the package.json for sample app, not the top-level one.

Reply:
I looked through the other apps and they did have their own dependencies and devDependencies, should I create a ticket to get that cleaned up for all apps?

Reply:
There were just few if that helps

Reply:
hmm, I don't think it matters for `devDependencies`, and for `dependencies` it looks like it used to be an empty set `{}` and you're adding first items here. So not sure if there's work needed to clean this up for other apps. Just make sure you move the fluentui dependencies you're adding to the sample app's package.json.